### ISSUE TITLE
Migration cop says it autocorrects, but it doesn't if the comment is enable

### ISSUE THREAD

**Issue by tas50** [rubocop-hq/rubocop#7814]
## Expected behavior

Cop should actually autocorrect rubocop comment

## Actual behavior

RuboCop shows that it autocorrects the code, but it does not.

```
/Users/tsmith/dev/the_world2/cisco-cookbook/files/default/vendor/gems/cisco_node_utils-1.7.0/spec/spec_helper.rb:90:20: C: [Corrected] Migration/DepartmentName: Department name is missing.
  # rubocop:enable Style:BlockComments
                   ^^^^^
```

## Steps to reproduce the problem

Try to autocorrect this code:

```
# rubocop:disable Style/BlockComments
=begin
  # Stuff
=end
# rubocop:enable Style:BlockComments
```

## RuboCop version

master

**Comment by koic** [comment]
This would be a false positive because `DepartmentName:CopName` is an unexpected format in the disabled comment. So, I think it should not be offensed, not auto-correction.

**Comment by tejasbubane** [comment]
> `DepartmentName:CopName` is an unexpected format

Using `DepartmentName:CopName` does not disable the cop. So I think `Migration/DepartmentName` should catch this typo & flag offense to let user know the copname is wrong?

Maybe we could improve message to something like `did you mean DepartmentName/CopName`?

**Comment by koic** [comment]
> `did you mean DepartmentName/CopName` ?

This is interesting. On the other hand, this cop's role is to complement a department name. Including typo detection in this cop could be a bit more complicated. IMHO, perhaps it would be better to provide a different feature for typo detection.

**Comment by tas50** [comment]
Thanks for the quick fix @koic
